LLSimpleCamera/LLSimpleCamera.m

```diff
@@ -527,6 +578,20 @@ - (void)setCameraPosition:(LLCameraPosition)cameraPosition
     
     self.videoCaptureDevice = device;
     self.videoDeviceInput = videoInput;
+
+    if (_position == LLCameraPositionFront) {
+        AVCaptureConnection *videoConnection = [_movieFileOutput connectionWithMediaType:AVMediaTypeVideo];
+        if ([videoConnection isVideoMirroringSupported]) {
+            [videoConnection setVideoMirrored:YES];
```

Comment:
Why are you setting mirrored to **YES** ? shouldn't we check the **self.mirror*** property?